spec/models/abilities/common_spec.rb

```diff
@@ -259,6 +259,11 @@
       it { should_not be_able_to(:destroy, own_investment_in_selecting_budget) }
       it { should_not be_able_to(:destroy, own_investment_in_balloting_budget) }
 
+      it { should be_able_to(:edit, own_investment_in_accepting_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_reviewing_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_selecting_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_balloting_budget) }
```

Comment:
Another test checking a user is not able to edit another author's proposal would be awesome! :smile:

Reply:
Sorry, I meant to **update** another author's proposal :smile:.